tests/ttnn/unit_tests/operations/eltwise/backward/test_backward_div.py

```diff
@@ -28,7 +28,7 @@
 @pytest.mark.parametrize(
     "round_mode",
     (
-        None,
+        "None",
```

Comment:
why was this necessary?

Reply:
In PyTorch, the div operation accepts the options None, "trunc", "floor" as valid parameters. However, "tanh" is enclosed in double quotes, while None is not. Since this is a string parameter, it would be more consistent and uniform if all options, including None, were represented within double quotes. 

In Pytorch , other string parameter ops like gelu has its inputs "none", "tanh" in quotes. Thought this change would maintain a standardised format across the API.

Reply:
will revert this change back and update the PR